### PR TITLE
Admin panel facade

### PR DESCRIPTION
**Pull request by martadiachok**

### CODE REVIEW

File: src/main/java/com/easyrest/facade/admin/AddModeratorAccountFacade.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    private CreateModeratorAccountForm clickAddModerator() {
+        leftSideMenu.clickModerators();
+        moderatorsPage_adminPanel.clickOnCreateModerator();
+        return new CreateModeratorAccountForm(driver);
+    }
+
+    public Moderators_AdminPanel submitAddModeratorForm(String name, String email, String phone, String password,
+                                                        String date, String month, String year) {
+        clickAddModerator();
+        createModeratorAccountForm.inputName(name);
```
Comment (ViktoriaMilkevich) [inline src/main/java/com/easyrest/facade/admin/AddModeratorAccountFacade.java:35]: you repeat input many times, maybe you should create one method for input

Comment (martadiachok) [reply src/main/java/com/easyrest/facade/admin/AddModeratorAccountFacade.java:35]: > you repeat input many times, maybe you should create one method for input

should change method in component, it is not in that pull request, decided to change all methods for input in next sprint

File: src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package com.easyrest.facade.admin;
+
+import com.easyrest.components.admin.ActionOnUser;
+import com.easyrest.components.admin.LeftSideMenu;
+import org.openqa.selenium.WebDriver;
+
+public class ActionOnUserFacade {
+    private WebDriver driver;
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java]: Do you need the driver as an instance variable?

Comment (martadiachok) [reply src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java]: > Do you need the driver as an instance variable?

yes

Comment (dmatsev) [reply src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java]: Marta, why do you need a driver on the facade?

**Review by dmatsev — APPROVED** [review]

File: src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java (hunk 2 above)
Comment (martadiachok) [reply src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java]: > Do you need the driver as an instance variable?

I reviewed, don t need driver, fix it

**Pull request merged**